File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[][]}
 */
var findDifference = function(nums1, nums2) {
    let result1=[]
    let unique=new Set([...nums1])
    let unique2=new Set([...nums2])
    let result2=[]
    
    for (i of unique){
        if (!unique2.has (i)){
            result1.push(i)
        }
    }
    for(j of unique2){
        if(!unique.has(j)){
            result2.push(j)
        }
    }
    return [result1,result2]

};

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution(object):
    def findDifference(self, nums1, nums2):
        set1=set(nums1)
        set2=set(nums2)
        result1=[]
        result2=[]
        for i in set1:
            if i not in set2:
                result1.append(i)
        for j in set2:
            if j not in set1:
                result2.append(j)
        return [result1,result2]
            